test/test_priority_stateful.py

```diff
@@ -0,0 +1,62 @@
+import random
+
+from hypothesis import note
+from hypothesis import strategies as st
+from hypothesis.stateful import RuleBasedStateMachine, rule
+
+import priority
+
+
+class PriorityQueue(RuleBasedStateMachine):
+
+    def __init__(self):
+        super(PriorityQueue, self).__init__()
+        self.tree = priority.PriorityTree()
+        self.stream_ids = [0]
```

Comment:
Ah, we could do that. This is a relic from an earlier version of the code where I had something like:

```python
def _select_stream(seed):
    random.seed(seed)
    return random.choice(self.stream_ids)
```

as a way to select which (existing) stream to block/unblock/remove – and `random.choice` fails on a set. But then I threw that away, so yeah, this can be a set now.